draft-ietf-scitt-architecture.md

```diff
@@ -158,59 +160,59 @@ Statement:
 : any serializable information about an Artifact. To help interpretation of Statements, they must be tagged with a media type (as specified in {{RFC6838}}).
 For example, a statement may represent a Software Bill Of Materials (SBOM) that lists the ingredients of a software Artifact, or some endorsement or attestation about an Artifact.
 
-Claim:
+Signed Statement:
 
-: an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Claims are encoded as COSE signed objects; the payload of the COSE structure contains the Statement.
+: an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Signed Statements are encoded as COSE signed objects; the payload of the COSE structure contains the issued Statement.
 
 Issuer:
 
-: an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer or an endorser.
+: an entity that creates Signed Statements about software Artifacts in the supply chain. An Issuer may be the owner or author of software Artifacts, or an independent third party such as a reviewer or an endorser.
```

Comment:
> ...independent third party such as a reviewer or an endorser.

I can see statements being made that are "negative", meaning a security company makes a statement the artifact has many critical vulnerabilities and should be replaced with version:foo.
Do we want to elaborate endorser to include opposed, or another term that reflects a negative statement? 
Or, do we want to generalize all statements, and leave it to the consumer and artifactType to define the schema of positive/negative?